app/views/events/_form.html.erb

```diff
@@ -1,61 +1,77 @@
+      <div class="col-lg-10">
+        <%= f.select(:game_mode, League.game_modes.map {|k, v| [k.humanize, v]},
+                     { include_blank: t('events.new.select_game_mode')},
+                     { class: 'form-control'}) %>
+      </div>
+    <% when 'Tournament' %>
+      <%= f.label :game_mode, class: 'control-label col-lg-2' %>
+      <div class="col-lg-10">
+        <%= f.select(:game_mode, Tournament.game_modes.map {|k, v| [k.humanize, v]},
+                     { include_blank: t('events.new.select_game_mode') },
+                     { class: 'form-control' }) %>
+      </div>
+    <% end %>
+  </div>-->
   <div class="form-group">
     <%= f.label :player_type, class: 'control-label col-lg-2' %>
     <div class="col-lg-10">
       <%= f.select(:player_type, Event.player_types.map {|k, v| [k.humanize, v]},
                    {include_blank: t('events.new.select_player_type')},
-                   {class: 'form-control'}) %>
+                   { class: 'form-control'}) %>
     </div>
   </div>
   <div class="form-group">
     <%= f.label :max_teams, class: 'control-label col-lg-2' %>
     <div class="col-lg-10">
-      <%= f.text_field :max_teams, id: :event_max_teams, class: 'form-control' %>
+      <%= f.text_field :max_teams, class: 'form-control' %>
       <%= f.error_span(:max_teams) %>
     </div>
   </div>
   <div class="form-group">
     <%= f.label :discipline, class: 'control-label col-lg-2' %>
     <div class="col-lg-10">
-      <%= f.text_field :discipline, id: :event_discipline, class: 'form-control' %>
+      <%= f.text_field :discipline, class: 'form-control' %>
       <%= f.error_span(:discipline) %>
     </div>
   </div>
   <div class="form-group">
     <%= f.label :deadline, class: 'control-label col-lg-2' %>
     <div class="col-lg-10">
-      <%= f.text_field :deadline, id: :event_deadline, class: 'form-control', data: { provide: 'datepicker', 'date-format': "yyyy-mm-dd" } %>
+      <%= f.text_field :deadline, class: 'form-control', data: {provide: 'datepicker', 'date-format' => 'yyyy-mm-dd'} %>
       <%= f.error_span(:deadline) %>
     </div>
   </div>
   <div class="form-group">
     <%= f.label :startdate, class: 'control-label col-lg-2' %>
     <div class="col-lg-10">
-      <%= f.text_field :startdate, id: :event_startdate, class: 'form-control', data: { provide: 'datepicker', 'date-format': "yyyy-mm-dd" } %>
+      <%= f.text_field :startdate, class: 'form-control', data: {provide: 'datepicker', 'date-format' => 'yyyy-mm-dd'} %>
       <%= f.error_span(:startdate) %>
     </div>
   </div>
   <div class="form-group">
     <%= f.label :enddate, class: 'control-label col-lg-2' %>
     <div class="col-lg-10">
-      <%= f.text_field :enddate, id: :event_enddate, class: 'form-control', data: { provide: 'datepicker', 'date-format': "yyyy-mm-dd" } %>
+      <%= f.text_field :enddate, class: 'form-control', data: {provide: 'datepicker', 'date-format' => 'yyyy-mm-dd'} %>
```

Comment:
Shouldn't it be possible to also parse a symbol here. like `date-format: ''yyyy-mm-dd"` ? instead of mixing strings and symbols? Or is there a specific reason to use one over the other?